OpenSim/Simulation/Model/Ligament.cpp

```diff
@@ -204,9 +202,28 @@ void Ligament::extendPostScale(const SimTK::State& s, const ScaleSet& scaleSet)
     }
 }
 
-const double& Ligament::getTension(const SimTK::State& s) const
+double Ligament::getTension(const SimTK::State& s) const
```

Comment:
`Ligament` was refactored to "properly" use cache variables.

Now, calling `getTension` will result in a cache variable computation (if necessary). Previously, the tension was computed during `computeForce`, which created a temporal dependency between `getTension` and `computeForce` (i.e. you couldn't use `Ligament::getTension` without first calling `Ligament::computeForce`).